README.md

```diff
@@ -0,0 +1,31 @@
+## About Eclipse Che Workspace Data Sync
+
+This tooling provide ability to increase I/O performance for a developer workspaces.
+
+==============
```

Comment:
Why it's here? Don't headers help to separate one section from another?

Reply:
fixed